jest.config.base.js

```diff
@@ -34,7 +34,7 @@ module.exports = {
   coverageProvider: 'v8',
 
   // A list of reporter names that Jest uses when writing coverage reports
-  coverageReporters: ['html', 'json-summary', 'text'],
+  coverageReporters: ['html', 'json-summary', 'text', 'json'],
```

Comment:
I don't think we need `json-summary`?
```suggestion
  coverageReporters: ['html', 'json', 'text'],
```

Reply:
`json-summary` is for `jest-it-up`